esrally/track/loader.py

```diff
@@ -1263,7 +1264,7 @@ def parse_task(self, task_spec, ops, challenge_name, default_warmup_iterations=N
                                               default_value=default_time_period),
                           clients=self._r(task_spec, "clients", error_ctx=op.name, mandatory=False, default_value=1),
                           # this will work because op_name must always be set, i.e. it is never `None`.
-                          completes_parent=(op.name == completed_by_name),
```

Comment:
Nit: I think we can get rid of the comment in the line above as well as it is not applicable anymore IMHO.